README.md

```diff
@@ -43,118 +55,524 @@ Check the original technical report for an earlier version of the Model Extensio
-| input_array             | [Array Object](#feature-array-object)                                            | **REQUIRED.** The N-dimensional array object that describes the shape, dimension ordering, and data type.                                                                                                                                          |     |
-| norm_by_channel         | boolean                                                                          | Whether to normalize each channel by channel-wise statistics or to normalize by dataset statistics. If True, use an array of [Statistics Objects](#bands-and-statistics) that is ordered like the `bands` field in this object.                    |     |
-| norm_type               | string                                                                           | Normalization method. Select one option from `min_max`, `z_score`, `max_norm`, `mean_norm`, `unit_variance`, `norm_with_clip`, `none`                                                                                                              |     |
-| resize_type             | string                                                                           | High-level descriptor of the rescaling method to change image shape. Select one option from `crop`, `pad`, `interpolation`, `none`. If your rescaling method combines more than one of these operations, provide the name of the operation instead |     |
-| statistics              | [Statistics Object](stac-statistics) `\|` [[Statistics Object](stac-statistics)] | Dataset statistics for the training dataset used to normalize the inputs.                                                                                                                                                                          |     |
-| norm_with_clip_values   | [integer]                                                                        | If `norm_type = "norm_with_clip"` this array supplies a value that is less than the band maximum. The array must be the same length as "bands", each value is used to divide each band before clipping values between 0 and 1.                     |
-| pre_processing_function | string                                                                           | A url to the preprocessing function where normalization and rescaling takes place, and any other significant operations. Or, instead, the function code path, for example: `my_python_module_name:my_processing_function`                          |     |
+### Framework
 
-### Accelerator Enum
+In most cases, this should correspond to common library names of well-established ML frameworks.
+No explicit "Enum" is defined to allow easy addition of newer frameworks, but it is recommended
+to use common names when applicable. Below are a few notable entries.
+
+- `PyTorch`
+- `TensorFlow`
+- `Scikit-learn`
+- `Huggingface`
+- `PyMC`
+- `JAX`
+- `ONNX`
+- `MXNet`
+- `Keras`
+- `Caffe`
+- `Weka`
```

Comment:
should `Weka` be listed? I've never heard of it or seen it in the wild.

Here's a suggested reordering based on my subjective interpretation of current popularity + longevity. 

I also added `rgee` and `spatialRF` to showcase some R options. especially in academia, lots of folks use R, particularly random forest models for semantic segmentation.

I removed Caffe (no updates in 4 years) and MxNet (archived last year). I don't think anyone will publish models for these frameworks.

Removed ONNX since it isn't a training framework and I think the purpose of this field is to describe the framework used to train the model. this might be different than the inference runtime and format.

```suggestion
- `PyTorch`
- `TensorFlow`
- `Scikit-learn`
- `Huggingface`
- `Keras`
- `rgee`
- `spatialRF`
- `JAX`
- `PyMC`
```